presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -401,8 +421,8 @@ void PrestoServer::run() {
             destination,
             queue,
             pool,
-            driverExecutor_.get(),
-            exchangeHttpExecutor_.get(),
+            exchangeHttpCpuExecutor_.get(),
```

Comment:
Do we see improvement with a dedicated cpu executor for this?

Reply:
When a Driver executor is busy doing computation it may result in increase in Exchange request latency, slowing down data exchange. The idea is that having a separate executor should decouple Exchange communication from execution potentially improving throughput.